upgrade_test.py

```diff
@@ -507,12 +506,15 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
             # upgrade first node
             self.db_cluster.node_to_upgrade = self.db_cluster.nodes[indexes[0]]
             self.log.info('Upgrade Node %s begin', self.db_cluster.node_to_upgrade.name)
-            self.upgrade_node(self.db_cluster.node_to_upgrade)
-            self.log.info('Upgrade Node %s ended', self.db_cluster.node_to_upgrade.name)
-            self.db_cluster.node_to_upgrade.check_node_health()
+            result = self.db_cluster.node_to_upgrade.remoter.run('scylla --version')
+            self.orig_ver = result.stdout
+            with ignore_upgrade_cdc_errors(self.orig_ver):
+                self.upgrade_node(self.db_cluster.node_to_upgrade)
+                self.log.info('Upgrade Node %s ended', self.db_cluster.node_to_upgrade.name)
+                self.db_cluster.node_to_upgrade.check_node_health()
 
-            # wait for the prepare write workload to finish
-            self.verify_stress_thread(prepare_write_cs_thread_pool)
+                # wait for the prepare write workload to finish
```

Comment:
Why this part  should  be also under the ignore  context manager?

Reply:
In one test, I saw the event was delayed for 3+ mins aftre upgrade_node() returned.

In the above code, the check_node_health() will finish very soon, the event might be raised in verify_stress_thread().